src/spec-lang/expr_grammar.pegjs

```diff
@@ -130,6 +151,7 @@ IF_SUCCEEDS = "if_succeeds"
 IF_UPDATED = "if_updated"
 IF_ASSIGNED = "if_assigned"
 DEFINE = "define"
+FORALL = "forall"
```

Comment:
Should we also add it to `Keyword` rule?

Reply:
Fixed in 2c86523